tests/test_scripts.py

```diff
@@ -1540,6 +1540,43 @@ def test_split_paired_reads_2_fq():
     assert n > 0
 
 
+def test_split_paired_reads_3_output_dir():
+    # test input file
+    infile = utils.get_test_data('paired.fq')
+
+    ex_outfile1 = utils.get_test_data('paired.fq.1')
+    ex_outfile2 = utils.get_test_data('paired.fq.2')
+
+    # actual output files...
+    outfile1 = utils.get_temp_filename('paired.fq.1')
+    in_dir = os.path.dirname(outfile1)
```

Comment:
would suggest renaming to output_dir - I think in_dir is confusing, because it's either "put data IN this dir" or "INput dir". Or 'into_dir'.